{"name": "Create order with invalid ingredient hash", "status": "passed", "description": "This test verifies that an order creation attempt with an invalid ingredient hash returns a 500 Internal Server Error.", "start": 1726410948854, "stop": 1726410949033, "uuid": "b838199d-872c-4816-a045-8ff5dd96aeea", "historyId": "70b892f8b0761e734d89f7532bbf49e8", "testCaseId": "70b892f8b0761e734d89f7532bbf49e8", "fullName": "tests.test_create_order.TestCreateOrder#test_create_order_with_invalid_ingredient_hash", "labels": [{"name": "suite", "value": "Order Creation Tests"}, {"name": "parentSuite", "value": "tests"}, {"name": "subSuite", "value": "TestCreateOrder"}, {"name": "host", "value": "DESKTOP-OODMP73"}, {"name": "thread", "value": "9352-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_create_order"}]}